### PR TITLE
Add SFTP/VSFTP documentation

### PR DESCRIPTION
**Pull request by fviale**
Also, add PARENT_JOB_ID generic info

### CODE REVIEW

File: src/docs/admin/references/properties/scheduler.properties [hunk 1]
```diff
@@ -264,6 +264,18 @@ pa.scheduler.auth.policy.refreshperiod.seconds=30
 #pa.scheduler.dataspace.defaultuser.localpath=
 #pa.scheduler.dataspace.defaultuser.hostname=
 
+# Is userspace based on impersonation (true) or path (false)
```
Comment (mbenguig) [inline src/docs/admin/references/properties/scheduler.properties:267]: Is ?

Comment (fviale) [reply src/docs/admin/references/properties/scheduler.properties:267]: the scheduler.properties file is added automatically from github scheduling. Modifications needs to be done inside the scheduling project if needed.

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 2]
```diff
@@ -351,6 +351,141 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+The user space will use the path defined in `pa.scheduler.dataspace.defaultuser.localpath` as its root and create sub-folders according to each user login name.
+
+When these properties are not configured, global and user spaces default locations are:
+
+ * Global Space : `PROACTIVE_HOME/data/defaultglobal`
+ * User Space : `PROACTIVE_HOME/data/defaultuser`
+
+==== SFTP protocol
+
+The SFTP protocol can be used instead of ProActive to deploy the User or Global spaces.
```
Comment (mbenguig) [inline src/docs/admin/ProActiveAdminGuide.adoc:397]: .. instead of ProActive protocols ...

File: src/docs/admin/ProActiveAdminGuide.adoc [hunk 3]
```diff
@@ -351,6 +351,141 @@ TIP: If you plan to use another DB version, it is important to get the correspon
...
+
+ProActive User Space mapped using SFTP has the following benefits:
+
+ * It allows to map each User Space to the *HOME directory* of the user on the SFTP server.
+ * It allows *file impersonation* : a file copied to the User Space will belong to the user and not to the ProActive server program.
```
Comment (mbenguig) [inline src/docs/admin/ProActiveAdminGuide.adoc:402]: ... belong to the user system account and not to the user ProActive server account.

**Review by mbenguig — APPROVED** [review]
Very small comments

**Review by tao-xinxiu — COMMENTED** [review]
Some minor suggestions.

**Review by tao-xinxiu — APPROVED** [review]

**Pull request merged**